### ISSUE TITLE
cannot autoinject into generic class

### ISSUE THREAD

**Issue by andaag** [lukaspili/Auto-Dagger2#1]
Produces this kind of inject statement:
void inject(BusViewPresenter<V, S> busViewPresenter);

Skipping <V, S> should work?

**Comment by lukaspili** [comment]
Not sure I understand. Can you provide a code example?

**Comment by andaag** [comment]
I can see very easily why that's not understandable.

Here's another try, but with code syntax:

```
void inject(BusViewPresenter<V, S> busViewPresenter);

Skipping <V, S> should work?
```

Sorry about that!

**Comment by lukaspili** [comment]
It is fixed in the 1.1 relase.
You can use

``` java
`@AutoInjector(value = MainActivity.class, parameterizedTypes = {String.class, String.class})`  
`@AutoExpose(value = MainActivity.class, parameterizedTypes = {String.class, String.class})`
```

However you proposition of skipping <V, S> in `void inject(BusViewPresenter<V, S> busViewPresenter);` would not work because Dagger2 forbids to inject raw type. So you need to specify the parameterized types through `@AutoInjector(parameterizedTypes = ...)`.

Also, beware that the groupID changed for the 1.1 release, it's now: **com.github.lukaspili.autodagger2**
